springboot3-todo/src/main/java/com/example/Application.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ImportRuntimeHints;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+@ImportRuntimeHints(NativeRuntimeHints.class)
```

Comment:
in springboot 2 we had the annotation `@NativeHint(options = "--enable-https")`  do we not need this hint now?

Reply:
From what I can see those annotations were removed in SB3. You can see there is a hint registrar for the other one. This one moved to `pom.xml`

```xml
			<plugin>
				<groupId>org.springframework.boot</groupId>
				<artifactId>spring-boot-maven-plugin</artifactId>
				<configuration>
					<image>
						<env>
							<BP_NATIVE_IMAGE>true</BP_NATIVE_IMAGE>
							<BP_NATIVE_IMAGE_BUILD_ARGUMENTS>--enable-https</BP_NATIVE_IMAGE_BUILD_ARGUMENTS>
						</env>
					</image>
				</configuration>
			</plugin>
```

that's what I found in the documentation

Reply:
Actually I think that's only relevant if using a container to build native image. Let me research a bit more

Reply:
Ah I have missed that in the pom

Reply:
I actually haven't been able to find a good answer to this anywhere. I posted this: https://stackoverflow.com/questions/75762202/how-do-i-enable-https-in-spring-boot-3-native-image

Reply:
Seems as if it isn't needed anymore. We don't need to do anything as spring-web now contains [Graal configuration metadata to enable HTTPS by default](https://github.com/spring-projects/spring-framework/blob/main/spring-web/src/main/resources/META-INF/native-image/org.springframework/spring-web/native-image.properties).

I'll revert this change in `pom.xml`.